.changeset/selfish-points-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Added: Show a count of fixable problems in the string formatter
```

Comment:
[suggest] Can you change the description for wording consistency in CHANGELOG.md?

```suggestion
Added: tally of fixable problems to `string` and `verbose` formatters
```